### PR TITLE
Provide timestamp of the last index run

### PR DESCRIPTION
**Pull request by BilelYahia**
Provide details on the last index run (issue #56)

### CODE REVIEW

File: src/HtmlGenerator/Utilities/Markup.cs [hunk 1]
```diff
@@ -236,6 +236,7 @@ public static void WriteLinkPanel(Action<string> writeLine, params string[] rows
                 writeLine(row);
             }
 
+            writeLine(string.Format("<tr><td>File indexed on: {0}</td></tr>", DateTime.Now.ToString()));
```
Comment (KirillOsenkov) [inline src/HtmlGenerator/Utilities/Markup.cs]: We must have a CSS style for gray somewhere, if not we should mark this string as gray via CSS.

Comment (KirillOsenkov) [inline src/HtmlGenerator/Utilities/Markup.cs]: Also I'm thinking it's better to have the "File indexed on" at the end of the source code, not on the bottom panel. This way we don't waste valuable space and people who want to know when the file was indexed can just scroll to the bottom.

File: src/SourceIndexServer/styles.css [hunk 2]
```diff
@@ -528,7 +528,7 @@ a i {
     background-color: aliceblue;
     padding: 8px;
     color: gray;
-    height: 50px;
+    height: 70px;
```
Comment (KirillOsenkov) [inline src/SourceIndexServer/styles.css]: I'm hesitant to increase the bottom panel as it's already a waste of space...

**Comment by jovianarts** [comment]
Just wanted to add my 2 cents. I found this idea interesting and wanted something similar some time ago but resolved it differently. I modified the banner template to include a token which get updated by the process that invokes the SourceBrowser executable. However, instead of the date at which **the process gets invoked or the HTML file generated**, it's the timestamp of the **latest commit I sync to**. That time is a good representation of the "snapshot" being generated and takes into account new commits that get submitted while my process is running that aren't going to be included, and avoiding confusion.

**Pull request closed without merging**